.github/workflows/tests.yml

```
@@ -7,12 +7,14 @@ env:
   HOMEBREW_DEVELOPER: 1
   HOMEBREW_GITHUB_ACTIONS: 1
   HOMEBREW_NO_AUTO_UPDATE: 1
+  HOMEBREW_CHANGE_ARCH_TO_ARM: 1
```

```suggestion
  HOMEBREW_CHANGE_ARCH_TO_ARM: 1
  HOMEBREW_REQUIRE_BOTTLED_ARM: 1
```

once https://github.com/Homebrew/homebrew-test-bot/pull/543 is merged. The `version` changes will still be required but the rest can be removed.